commands.go

```diff
@@ -462,10 +473,17 @@ func doFetch(c *cli.Context) error {
 		os.Exit(1)
 	}
 
-	metricValues, err := newMackerelFromContext(c).FetchLatestMetricValues(argHostIDs, optMetricNames)
-	logger.DieIf(err)
+	allMetricValues := make(mkr.LatestMetricValues)
+	// Fetches 100 hosts per one request (to avoid URL maximum length).
+	for _, hostIds := range split(argHostIDs, 100) {
+		metricValues, err := newMackerelFromContext(c).FetchLatestMetricValues(hostIds, optMetricNames)
+		logger.DieIf(err)
+		for metricName := range metricValues {
```

Comment:
[nit]
The key of `LatestMetricValues` is actually hostId, not `metricName`.
(As long as mkr and mackerel-client-go's definition, it's perfectly unknown what key of `LatestMetricValues` is, so imo it's good to use some ambiguous name like `key`...)

Reply:
For sure.